src/verb-plugin.js

```diff
@@ -42,7 +42,7 @@ const isVerb = async word => {
   try {
     definitions = await wordnet.lookup(word);
   } catch (err) {
-    return whitelistedVerbs.includes(word);
+    return null;
```

Comment:
Hmm, should this return null or false?

Reply:
Should be false, will change.

Reply:
Actually I will close this PR. #7 Changes the strategy.